test/e2e/kubectl/kubectl.go

```diff
@@ -419,110 +415,6 @@ var _ = SIGDescribe("Kubectl client", func() {
-				"IngressClass": true,
-
-				// ignored temporarily while waiting for bug fix.
-				"ComponentStatus":    true,
-				"ClusterRoleBinding": true,
-				"ResourceQuota":      true,
-
-				// ignored because no test data exists.
-				// Do not add anything to this list, instead add fixtures in
-				// the test/integration/etcd package.
-				"BackendConfig":         true,
-				"NodeMetrics":           true,
-				"PodMetrics":            true,
-				"VolumeSnapshotClass":   true,
-				"VolumeSnapshotContent": true,
-				"VolumeSnapshot":        true,
-			}
-
-			apiGroups, err := c.Discovery().ServerPreferredResources()
-
-			if discovery.IsGroupDiscoveryFailedError(err) {
-				discoveryErr := err.(*discovery.ErrGroupDiscoveryFailed)
-				for gv := range discoveryErr.Groups {
-					if strings.Contains(gv.Group, ".") && !strings.HasSuffix(gv.Group, ".k8s.io") {
-						// tolerate discovery errors for non-k8s.io groups (like aggregated/crd groups)
-						continue
-					}
-					if gv.Group == "wardle.k8s.io" || gv.Group == "metrics.k8s.io" {
-						// tolerate discovery errors for known test k8s.io groups like aggregated/metrics groups
-						continue
-					}
-					// otherwise, fail
-					framework.ExpectNoError(err)
-				}
-			} else {
-				// fail immediately if this isn't a discovery error
-				framework.ExpectNoError(err)
-
-			}
-
-			testableResources := etcd.GetEtcdStorageDataForNamespace(f.Namespace.Name)
-
-			for _, group := range apiGroups {
-				// This limits the scope of this test to exclude CRDs. This
-				// assumes that CRDs will not have a .k8s.io group and will have
-				// a . in their name.
-				if !strings.Contains(group.GroupVersion, ".k8s.io") && strings.Contains(group.GroupVersion, ".") {
-					continue
-				}
-
-				for _, resource := range group.APIResources {
-					if !verbsContain(resource.Verbs, "get") || ignoredResources[resource.Kind] || strings.HasPrefix(resource.Name, "e2e-test") {
-						continue
-					}
-
-					// compute gvr
-					gv, err := schema.ParseGroupVersion(group.GroupVersion)
-					framework.ExpectNoError(err)
-					gvr := gv.WithResource(resource.Name)
-
-					// assert test data exists
-					testData := testableResources[gvr]
-					gomega.ExpectWithOffset(1, testData).ToNot(gomega.BeZero(), "No test data available for %s", gvr)
-
-					// create test resource
-					mapping := &meta.RESTMapping{
-						Resource:         gvr,
-						GroupVersionKind: gv.WithKind(resource.Kind),
-					}
-
-					if resource.Namespaced {
-						mapping.Scope = meta.RESTScopeNamespace
-					} else {
-						mapping.Scope = meta.RESTScopeRoot
-					}
-
-					client, obj, err := etcd.JSONToUnstructured(testData.Stub, f.Namespace.Name, mapping, f.DynamicClient)
-					framework.ExpectNoError(err)
-					if resource.Kind != "APIService" && resource.Kind != "CustomResourceDefinition" {
-						obj.SetName(obj.GetName() + randString)
-					}
-
-					createObjValidateOutputAndCleanup(client, obj, resource)
```

Comment:
static check will complain about createObjValidateOutputAndCleanup and verbsContain being unused

Reply:
Thanks for catching that, I've removed those + the deleteObj helper function used by this test.